appveyor.yml

```diff
@@ -0,0 +1,101 @@
+environment:
+  github_release_token:
+    secure: en/UnoFHA3RsSAHM5c6zb3RAMa4V01lC3pYG1YrHSeOPFu78SUA/0lOjNe3pp8RG
+  matrix:
+
+      # Node v0.10, MSVS 2013
+    - nodejs_version: 0.10.40
+      nodejs_abi: 11
+      platform: x86
+      target_arch: ia32
+      msvs_version: 2013
+
+    - nodejs_version: 0.10.40
+      nodejs_abi: 11
+      platform: x64
+      target_arch: x64
+      msvs_version: 2013
+
+      # Node v0.12, MSVS 2015
+    - nodejs_version: 0.12.7
+      nodejs_abi: 14
+      platform: x86
+      target_arch: ia32
+      msvs_version: 2015
+
+    - nodejs_version: 0.12.7
+      nodejs_abi: 14
+      platform: x64
+      target_arch: x64
+      msvs_version: 2015
+
+      # Node v0.12, MSVS 2013
+    - nodejs_version: 0.12.7
+      nodejs_abi: 14
+      platform: x86
+      target_arch: ia32
+      msvs_version: 2013
+
+    - nodejs_version: 0.12.7
+      nodejs_abi: 14
+      platform: x64
+      target_arch: x64
+      msvs_version: 2013
+
+      # Node v4, MSVS 2013
+    #- nodejs_version: 4.0.0
+    #  nodejs_abi: 46
+    #  platform: x86
+    #  target_arch: ia32
+    #  msvs_version: 2013
```

Comment:
node-gyp isn't running in AppVeyor's 32bit version with Node 4 and 5. I believe it has something to do with downloading NodeJS header files.